presto-server-rpm/src/main/rpm/preremove

```diff
@@ -0,0 +1,6 @@
+# Pre erase script
+
+if ! /etc/init.d/presto status | grep -q 'Not running'; then 
```

Comment:
Do we need to grep for a message? Per the LSB standard, can we simply check for status 3?

http://refspecs.linuxbase.org/LSB_3.0.0/LSB-PDA/LSB-PDA/iniscrptact.html